tools/kontaktdaten.py

```diff
@@ -113,9 +186,9 @@ def validate_phone(phone):
     if not isinstance(phone, str):
         raise ValidationError("Muss eine Zeichenkette sein")
 
-    if not re.match(r"^(\+49)?[0-9]+$", phone):
+    if not re.match(r"^(\+49){1}[0-9]+$", phone):
```

Comment:
`{1}` ist unnötig. Man kann das hier vielleicht wie folgt verbessern; ist aber nicht getestet.

```suggestion
    if not re.match(r"^\+49[1-9][0-9]+$", phone):
```